src/NServiceBus.Core.Tests/Pipeline/PipelineTests.cs

```diff
@@ -22,6 +22,11 @@ public class PipelineTests
         [Test]
         public async Task ShouldExecutePipeline()
         {
+            if (Environment.OSVersion.Platform != PlatformID.Win32NT)
```

Comment:
NUnit has a Platform attribute that I tried to use first, bu  it doesn't exist in their current netstandard assemblies. I suspect it's because it relies on the above code behind the scenes, and that API was added back to .NET Core in 2.0.